### PR TITLE
Add myself as maintainer in hooks

### PR DESCRIPTION
**Pull request by pvinis**
# Summary
Adding myself as maintainer in hooks

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 | [react-native-geolocation](https://github.com/react-native-community/react-native-geolocation) | [@janicduplessis](https://github.com/janicduplessis), [@matt-oakes](https://github.com/matt-oakes), [@michalchudziak](https://github.com/michalchudziak) | active |
 | [react-native-google-analytics](https://github.com/react-native-community/react-native-google-analytics) | [@brentvatne](https://github.com/brentvatne) | inactive(?) |
 | [react-native-google-signin](https://github.com/react-native-community/react-native-google-signin) | [@vonovak](https://github.com/vonovak) | active |
-| [react-native-hooks](https://github.com/react-native-community/react-native-hooks) | [@dabit3](https://github.com/dabit3), [@melihberberolu](https://github.com/melihberberolu) | active |
+| [react-native-hooks](https://github.com/react-native-community/react-native-hooks) | [@dabit3](https://github.com/dabit3), [@melihberberolu](https://github.com/melihberberolu), [@pvinis](https://github.com/pvinis) | active |
```
Comment (lucasbento) [inline MAINTAINERS.md]: Do you know if the others are still active?

Comment (pvinis) [reply MAINTAINERS.md]: Let's ping and ask. @dabit3 @melihberberolu

Comment (LinusU) [reply MAINTAINERS.md]: As discussed in Discord I would also like to be a maintainer on this package, should I add myself in a follow up PR? ☺️

Comment (lucasbento) [reply MAINTAINERS.md]: Oh yeah, please! I'll send you an invite for the organisation.

Comment (pvinis) [reply MAINTAINERS.md]: I can add you in this PR, and then once we know the status of the order two, we can merge.

Comment (LinusU) [reply MAINTAINERS.md]: Sounds perfect 👍

**Review by lucasbento — APPROVED** [review]
Let's merge this as it is, we can edit it later.

Thanks @pvinis & @LinusU ❤️

**Pull request merged**